samples/wifi/thread_coex/README.rst

```diff
@@ -0,0 +1,400 @@
+
+Configuration
+*************
+
+|config|
+
+Configuration options
+=====================
+
+The following sample-specific Kconfig options are used in this sample (located in :file:`samples/wifi/thread_coex/Kconfig`):
+
+
+.. _CONFIG_TEST_TYPE_WLAN_ONLY:
+
+CONFIG_TEST_TYPE_WLAN_ONLY
+   This option enables the Wi-Fi test type.
+
+.. _CONFIG_TEST_TYPE_OT_ONLY:
+
+CONFIG_TEST_TYPE_OT_ONLY
+   This option enables the Thread test type.
+
+.. _CONFIG_TEST_TYPE_WLAN_OT:
+
+CONFIG_TEST_TYPE_WLAN_OT
+   This option enables concurrent Wi-Fi and Thread tests.
+
+.. _CONFIG_COEX_TEST_DURATION:
+
+CONFIG_COEX_TEST_DURATION
+   This option sets the Wi-Fi, Thread, or both test duration in milliseconds.
+
+.. _CONFIG_STA_SSID:
+
+CONFIG_STA_SSID
+   This option specifies the SSID of the Wi-Fi access point to connect.
+
+.. _CONFIG_STA_PASSWORD:
+
+CONFIG_STA_PASSWORD
+   This option specifies the Wi-Fi passphrase (WPA2™) or password (WPA3™) to connect.
+
+.. _CONFIG_STA_KEY_MGMT_*:
+
+CONFIG_STA_KEY_MGMT_*
+   These options specify the Wi-Fi key security option.
+
+Additional configuration
+========================
+
+To enable different test modes, set up the following configuration parameters in the :file:`prj.conf` file:
+
+* Test modes - Use the following Kconfig options to select the required test case:
+  * :ref:`CONFIG_TEST_TYPE_WLAN_ONLY <CONFIG_TEST_TYPE_WLAN_ONLY>` for Wi-Fi-only test.
+  * :ref:`CONFIG_TEST_TYPE_OT_ONLY <CONFIG_TEST_TYPE_OT_ONLY>` for Thread-only test.
+  * :ref:`CONFIG_TEST_TYPE_WLAN_OT <CONFIG_TEST_TYPE_WLAN_OT>` for concurrent Wi-Fi and Thread test.
+
+* Test duration - Use the :ref:`CONFIG_COEX_TEST_DURATION <CONFIG_COEX_TEST_DURATION>` Kconfig option to set the duration of the Wi-Fi-only test or Thread-only test or both.
+  The units are in milliseconds.
+  For example, to set the test for 20 seconds, set this value to ``20000``.
+
+* Wi-Fi connection - Set the following options appropriately as per the credentials of the access point used for this testing:
+
+.. include:: /includes/wifi_credentials_static.txt
+
+.. note::
+   ``menuconfig`` can also be used to configure ``Wi-Fi credentials``
+
+* Wi-Fi throughput test - Set the :kconfig:option:`CONFIG_NET_CONFIG_PEER_IPV4_ADDR` Kconfig option appropriately as per the IP address of the test PC on which **iperf** is run.
+
+See :ref:`zephyr:menuconfig` in the Zephyr documentation for instructions on how to run ``menuconfig``.
+
+Building and running
+********************
+
+.. |sample path| replace:: :file:`samples/wifi/thread_coex`
+
+.. include:: /includes/build_and_run_ns.txt
+
+You can build the sample for the following configurations:
+
+* Wi-Fi throughput only
+* Thread throughput only
+* Concurrent Wi-Fi and Thread throughput (with coexistence enabled and disabled modes)
+
+To build for the nRF7002 DK, use the ``nrf7002dk/nrf5340/cpuapp`` build target.
```

Comment:
Should build target >>**board** target here and at all instances?
Related doc page in Zephyr: https://docs.nordicsemi.com/bundle/ncs-latest/page/zephyr/hardware/porting/board_porting.html#board_terminology

Reply:
No. For example, as per the terminology bl5340_dvk@1.2.0/nrf5340/cpuapp/ns is called board target. nrf5340/cpuapp/ns is called board qualifier.